.github/workflows/github.yml

```diff
@@ -337,17 +416,35 @@ jobs:
             gzip release.tar
             
         - name: Upload source code archive
+          id: upload_source
           uses: actions/upload-artifact@v4
           with:
             name: ${{ env.VCMI_PACKAGE_FILE_NAME }}
-            compression-level: 0
+            compression-level: 9
```

Comment:
Same here. Tar.gz is already compressed and with the same algorithm

Reply:
No. Tar.gz (or tar.whatever) is default format for source tarballs. And .zip is enforced by Github.
When publishing release I extract .zip's of each artifact and add its content to release.
Heck, we now have zip in zip! Cmake generates .zip installer for Windows and Github packs it into another zip when uploading artifact.

But let's keep it as it for consistency.

Reply:
Yes, ZIP in ZIP will needs to be solved in future for OpenSign. Already added corresponding comments to CMakeLists.
This can be done in future CI adjustments.